templates/postgresql.conf.j2

```diff
@@ -68,7 +68,6 @@ temp_buffers                    = {{ postgres_temp_buffers }}
 max_prepared_transactions       = {{ postgres_max_prepared_transactions }}
 work_mem                        = {{ postgres_work_mem }}
 maintenance_work_mem            = {{ postgres_maintenance_work_mem }}
-replacement_sort_tuples         = {{ postgres_replacement_sort_tuples }}
```

Comment:
Did postgres remove support for all these vars? There are certainly still defaults for them.
Maybe remove the defaults and add a check whether the var is set instead (if you don't want to have them in the config in all cases).

Reply:
postgres adds support for those vars in further version, removed them in the defaults

Reply:
this var is still in the defaults, otherwise it's fine